standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5150,6 +5166,7 @@ private int abortTxns(Connection dbConn, List<Long> txnids, boolean checkHeartbe
       if (MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.METASTORE_ACIDMETRICS_EXT_ON)) {
         Metrics.getOrCreateCounter(MetricsConstants.TOTAL_NUM_ABORTED_TXNS).inc(txnids.size());
       }
+      LOG.warn("Txns are aborted successfully due to : {}. Aborted Transaction IDs : {}", errorMsg, txnids);
```

Comment:
````
LOG.info("Aborted transactions {} due to {}", txnids, errorMsg);
````
errorMsg should be just text, like "heartbeat time-out"
either update the msg in TxnErrorMsg or use it as pattern for the string formatter
````
ABORT_ROLLBACK(50005, "<<<Txn aborted due to>>> rollback"),
````

Reply:
Used this format of logging as suggested - 
`LOG.info("Aborted transactions {} due to {}", txnids, errorMsg);`

Updated the error msgs to make it concise in TxnErrorMsg.